Lib/pipes.py

```diff
@@ -48,8 +48,8 @@
 exit status of the conversion pipeline.
 
 To open a file for reading or writing through a conversion pipeline:
-   fp = t.open(file, mode)
-where mode is 'r' to read the file, or 'w' to write it -- just like
+   fp = t.open(file, rw)
+where rw is 'r' to read the file, or 'w' to write it -- just like
```

Comment:
```suggestion
where 'rw' is 'r' to read the file, or 'w' to write it -- just like
```

Reply:
Hmm, I wonder if making it a string could be misleading - without quotes, it's clear that it's a variable of some sort, while with, maybe it gets confused as a value

Reply:
A compromise could be:

```suggestion
where *rw* is 'r' to read the file, or 'w' to write it -- just like
```

But the way @slateny has it currently is in keeping with the format used in the rest of the docstring: `infile` and `outfile` on line 46 are not surrounded by quotes or asterisks.

Reply:
Ah asterisks are a good idea, and good point on the consistency. I'm fine with both so I'll defer the choice here